flask_restplus/api.py

```diff
@@ -451,7 +451,8 @@ def specs_url(self):
 
         :rtype: str
         '''
-        return url_for(self.endpoint('specs'), _external=True)
+        _scheme = request.headers.get('x-forwarded-proto', 'http')
+        return url_for(self.endpoint('specs'), _external=True, _scheme=_scheme)
```

Comment:
I just hit the same issue and resolved it with monkey patching `_scheme=''`, similarly to the suggestion in #565. What's the advantage of using the approach in this PR versus setting `_scheme=''` to get a protocol-relative URL?